### ISSUE TITLE
Hacktoberfest celebration: Blogs needed for the website (Special Prize)

### ISSUE THREAD

**Issue by akshitagupta15june** [akshitagupta15june/PetMe#60]
Write a blog on anything related to pet or stray animals:

for example if you have a pet in your home you can write how you feel as a pet owner, if you have helped any stray animals what was your experience? these are just the example any blog related to stray animals and pets is acceptable.

You can choose any platform to write blog but we will be needing your content here for our section blogs.

The final part the best blog will receive a special prize, let me disclose it will be 

*Github Sticker bundle*🥳

![image](https://user-images.githubusercontent.com/57909583/193083801-712e838c-81b2-4cc7-a9e0-62c8d2f46021.png)

**Comment by akshitagupta15june** [comment]
After writing the blogs you can paste your links here!

**Comment by Anuran12** [comment]
Hey @akshitagupta15june I would like to write a blog about this. But where should I write that ?

**Comment by akshitagupta15june** [comment]
Any platform i would suggest medium or if you want you can write in Google docs and mail the pdf to akshitagupta15june@gmail.com

**Comment by Stk-79** [comment]
I would like to take up on this . Is there a deadline or something to send blog?

**Comment by akshitagupta15june** [comment]
Deadline 30 October 2022

**Comment by Stk-79** [comment]
> Deadline 30 October 2022

OK.

**Comment by krupalitrivedi** [comment]
Hi! I can write a blog :)

**Comment by akshitagupta15june** [comment]
Yes go for it
This issue is for everyone

**Comment by agarwaldevesh374** [comment]
Hi, please assign me this issue.

**Comment by akshitagupta15june** [comment]
Anyone can work on it

**Comment by omk-coder** [comment]
recently I feel like writing a blog on something but you just gave an  topic.  Thanks for this issue.

**Comment by akshitagupta15june** [comment]
Thanks waiting for your blog

**Comment by akshitagupta15june** [comment]
All the blogs will be published in the website

**Comment by danschenker** [comment]
At your request, I'd like to be a part of this one as well, please @akshitagupta15june 🙂

**Comment by akshitagupta15june** [comment]
yes anyone can work on this, all blogs will be published in this website

**Comment by akshitagupta15june** [comment]
@danschenker can u look at current issue #132

**Comment by danschenker** [comment]
> 

Just emailed a blog post to you via Google Docs!

**Comment by akshitagupta15june** [comment]
Thankyou so much
I will surely look into this and add it in website

**Comment by akshitagupta15june** [comment]
Waiting for blogs from all contributors!
deadline is 30 october 2022
After writing the blogs you can paste your links here! or if you want you can write in Google docs and mail the pdf to [akshitagupta15june@gmail.com](mailto:akshitagupta15june@gmail.com)
for discussion come at https://github.com/akshitagupta15june/PetMe/discussions/61

**Comment by LittleMissButtons** [comment]
Hiya. If we email you a copy of our blog post does that go towards our Hacktobefest count or do we need to do it via a pull request for it to count? Cheers

**Comment by EslemOuederni** [comment]
Here is my Google docs pet blog ! [](https://docs.google.com/document/d/1ttkonrd7inrBiKTDADwAyL-D5-bLUprrrvlepM8Rcio/edit?usp=sharing)

**Comment by akshitagupta15june** [comment]
thanks @EslemOuederni

**Comment by akshitagupta15june** [comment]
yes, u can choose any platform but we will be needing your content for our website blog so just mail me the doc content at akshitagupta15june@gmail.com

**Comment by akshitagupta15june** [comment]
yes, u can choose any platform but we will be needing your content for our website blog so just mail me the doc content at [akshitagupta15june@gmail.com](mailto:akshitagupta15june@gmail.com)

**Comment by wvinson43** [comment]
and I will email it to you as well!

**Comment by akshitagupta15june** [comment]
thankyou so much @wvinson43 for ur blog
waiting for others blog

**Comment by akshitagupta15june** [comment]
There are many open issues in repo you can contribute in them and don't forget to add your handle in contributors.md and give me a pr

**Comment by danschenker** [comment]
> You are welcome! How do I get a pull request credit for this? I am a first-timer to Hacktoberfest!

I'm trying to submit a PR for this as well, but I can't seem to do it on my fork without any code changes. Any suggestions @akshitagupta15june?

**Comment by akshitagupta15june** [comment]
yes you just have to share your blog with me at akshitagupta15june@gmail.com and then add your contribution details in contribution.md @danschenker

**Comment by danschenker** [comment]
Thanks for the quick response! I was already added to `contributors.md`, so I created contribution.md with a copy of my post from Google Docs. @akshitagupta15june

**Comment by akshitagupta15june** [comment]
ok fine then no issue i have got it and we are developing blogs.html, if you want to give pr then i will assign you a new issue where you can add your blog directly to the website by giving the pr
thanks
@danschenker

**Comment by danschenker** [comment]
> ok fine then no issue i have got it and we are developing blogs.html, if you want to give pr then i will assign you a new issue where you can add your blog directly to the website by giving the pr
> 
> thanks
> 
> @danschenker 

Ah okay sure, that should work! Thanks for that

**Comment by GingerKiwi** [comment]
Hi,
I just finished this post. I'm done for the day and will follow the process to get this credited as a Hacktoberfest PR tomorrow.

[Tips For New Feral Rescue Kitten Owners — Part 1 of 3
By Elizabeth McCready and rescue kitten Maybel](https://gingerkiwi-dev.medium.com/bc6d21da2dd9)

**Comment by akshitagupta15june** [comment]
great thanks @GingerKiwi! this blog will be added soon, till then please add your handle in contributors.md and give me a pr

**Comment by akshitagupta15june** [comment]
deadline is 30 october 2022
don't forget to mail your blogs

**Comment by Anuran12** [comment]
@akshitagupta15june i sent you today.. can i write more than one blog ?

**Comment by akshitagupta15june** [comment]
Sorry but for this competition only 1 blog is acceptable

But after 30 oct you can add

**Comment by omk-coder** [comment]
where should I paste the link here??

https://omk.hashnode.dev/all-about-pet-ownership-love-caring-helping-stray-dogs

**Comment by akshitagupta15june** [comment]
Thanks for your blog
Can u tell your full name

**Comment by omk-coder** [comment]
omkar ramishte.

**Comment by akshitagupta15june** [comment]
Thankyou so much

**Comment by omk-coder** [comment]
and just now I forked this repo again and the site looks amazing. UI changed

**Comment by akshitagupta15june** [comment]
Yes these are all the efforts of contributors like u.

**Comment by GingerKiwi** [comment]
Elizabeth McCready

https://github.com/GingerKiwi

Apologies for the delay. I've been helping my mom recover from Hurricane 
Fiona.

On 2022-10-07 07:19, AKSHITA GUPTA wrote:

> Thanks for your blog
> Can u tell your full name
> 
> --
> Reply to this email directly, view it on GitHub [1], or unsubscribe 
> [2].
> You are receiving this because you were mentioned.Message ID: 
> ***@***.***>



Links:
------
[1] 
https://github.com/akshitagupta15june/PetMe/issues/60#issuecomment-1271461793
[2]

**Comment by Omkar0803** [comment]
Can I contribute to this?

**Comment by akshitagupta15june** [comment]
Yes anyone can contribute

**Comment by akshitagupta15june** [comment]
We are waiting blogs from all contributors and currently @Anuran12 is adding all blogs received from all wonderful contributors to the main website PetMe........ Stay Tuned!

Thankyou everyone for the blogs and we are waiting for some more.

**Comment by TannisthaBiswas** [comment]
Hi, I would like to write a blog too.

**Comment by akshitagupta15june** [comment]
go for it @TannisthaBiswas

**Comment by AKG1301** [comment]
Abhinav K Gupta
Github: https://github.com/AKG1301
Hi, I would like to write a blog too.
Pdf Link: https://drive.google.com/file/d/1f2V9i2Pj56WZ_NOSjTyNITto3mmw4DtS/view?usp=sharing
[Blog on Dogs by abhinav K gupta.pdf](https://github.com/akshitagupta15june/PetMe/files/9758779/Blog.on.Dogs.by.abhinav.K.gupta.pdf)

**Comment by akshitagupta15june** [comment]
Thankyou for your blog

**Comment by TannisthaBiswas** [comment]
> go for it @TannisthaBiswas

https://docs.google.com/document/d/1LaDh1gSalIrFSt_wnN5YSICgaPku2-Xjsuc6W3rNn0Q/edit?usp=sharing Link for the blog. I've also mailed it to you. Do I add my name to contributors.md and send a PR now?

**Comment by akshitagupta15june** [comment]
Thanks for your blog

**Comment by Anuran12** [comment]
Hii everyone !!! The blog pages are published on the website... 

Here is the link https://akshitagupta15june.github.io/PetMe/blog.html

Check your blogs and there can be some human errors.. feel free to create an issue if you find any error..

**Comment by AKG1301** [comment]
please see this pr also.
https://github.com/akshitagupta15june/PetMe/issues/60#issuecomment-1275099388

**Comment by akshitagupta15june** [comment]
Yes on process
All blogs will be added

**Comment by Nikita0509** [comment]
@akshitagupta15june I would like to write a blog on my pet

**Comment by azulverdosa** [comment]
Hi there, 

I've just emailed you my blog post, will be adding my handle in contributors.md and sending you a PR next.

Thanks!

**Comment by LittleMissButtons** [comment]
Hi @akshitagupta15june I have just messaged you my blog post. It's about our rescue rooster so hopefully add a bit more variety.
Cheers!

**Comment by akshitagupta15june** [comment]
@Nikita0509 go for it!
Thanks @LittleMissButtons and @azulverdosa 
We will surely add your blogs.

Thanks for contributing 💜

**Comment by akshitagupta15june** [comment]
Thankyou so much @Anuran12 for the contributions you made in this project 💜

**Comment by akshitagupta15june** [comment]
Deadline to submit blogs is 30 October 2022
@Susmita-Dey is gonna review all blogs.
P.S all blogs will be published in the website 💥

**Comment by akshitagupta15june** [comment]
I would request all the blogs contributors to go to the website and looks at blogs.html page if you find any issues in your own blogs so you can create issue regarding your own blogs and can create a pull request for the same.
If you want to add some pictures, feel free to add ,but in your own blogs please 💜

@danschenker @LittleMissButtons @EslemOuederni @Anuran12 @omk-coder @TannisthaBiswas

**Comment by omk-coder** [comment]
sure.
I just  gone through the blog site  its looks attractive.
Their are some minor issue I like to tell.
1. toggle theme logo is not present.
2. 2 kafka's blog are their with same content.
3.  its not something I would pointout but inbox spelling is wrong at the end.
That's it no other issue are their its looks great.

**Comment by pagsum** [comment]
how can i contribute to this blog writing for hacktoberfest22?

**Comment by akshitagupta15june** [comment]
Everything is mentioned above @pagsum

**Comment by pagsum** [comment]
hey there i have added pagsum.html in blog folder.  please put my blog in blog.html webpage

**Comment by LittleMissButtons** [comment]
> 

Hiya, was there anything else I needed to do other than emailing you my blog to link it to the Hacktoberfest?

**Comment by AKG1301** [comment]
Hello, when i click on my Profile it will give 404 error. please link my profile correctly :).
Link of Blog :https://akshitagupta15june.github.io/PetMe/blogs/abhinav.html 
when i click on my profile it will give error . My github profile is: https://github.com/AKG1301

**Comment by akshitagupta15june** [comment]
> Hello, when i click on my Profile it will give 404 error. please link my profile correctly :). Link of Blog :https://akshitagupta15june.github.io/PetMe/blogs/abhinav.html when i click on my profile it will give error . My github profile is: https://github.com/AKG1301

@Anuran12

**Comment by Anuran12** [comment]
> Hello, when i click on my Profile it will give 404 error. please link my profile correctly :). Link of Blog :https://akshitagupta15june.github.io/PetMe/blogs/abhinav.html when i click on my profile it will give error . My github profile is: https://github.com/AKG1301

@AKG1301  solved.. please check

**Comment by AKG1301** [comment]
@Anuran12 @akshitagupta15june done.
Thank you 😊

> > Hello, when i click on my Profile it will give 404 error. please link my profile correctly :). Link of Blog :https://akshitagupta15june.github.io/PetMe/blogs/abhinav.html when i click on my profile it will give error . My github profile is: https://github.com/AKG1301
> 
> @AKG1301 solved.. please check

**Comment by anjaliSilawat** [comment]
I want to add more blogs and also want to improve the UI of blog section @akshitagupta15june I would like to work on this issue please assign it to me

**Comment by snehalsaurabh** [comment]
Can we still add to this issue?

**Comment by projectile07** [comment]
Hi there,
Is this issue still open? Would I still contribute!!

**Comment by anushka9555** [comment]
Hello. Can we still write the blogs? If yes, then I am interested

**Comment by projectile07** [comment]
> Hello. Can we still write the blogs? If yes, then I am interested

I'm hoping to get an issue assigned to me, so if the answer is yes, you can send me your blog and I'll update it to the website:)

**Comment by itssiddhantjain** [comment]
Hey @akshitagupta15june can I contribute?

**Comment by kamalsritha** [comment]
Is this still open, can you please assign me this issue under SWOC

**Comment by VIDHISHAK15** [comment]
Hey Akshita! I would like to write the Blog I am also a collaborator in SWOC S4 . Where can I write the blog?

**Comment by akshitagupta15june** [comment]
Go for it @EshaalakshmiDS

**Comment by lovelymahor** [comment]
I would like to work on this project. Please assign it to me!

**Comment by Gokilp** [comment]
please assign to me #gssoc24

**Comment by anu906** [comment]
@akshitagupta15june 
Can you assign this task to me.
Looking forward to work on it!

**Comment by real99riddhi** [comment]
Hi @akshitagupta15june can we send more blogs for this ? I'm interested in this topic.. ?

**Comment by piyushsahujtp** [comment]
> Write a blog on anything related to pet or stray animals:
> 
> for example if you have a pet in your home you can write how you feel as a pet owner, if you have helped any stray animals what was your experience? these are just the example any blog related to stray animals and pets is acceptable.
> 
> You can choose any platform to write blog but we will be needing your content here for our section blogs.
> 
> The final part the best blog will receive a special prize, let me disclose it will be
> 
> _Github Sticker bundle_🥳
> 
> After writing the blogs you can paste your links here! or if you want you can write in Google docs and mail the pdf to [akshitagupta15june@gmail.com](mailto:akshitagupta15june@gmail.com) for discussion come at #61
> 
> ![image](https://user-images.githubusercontent.com/57909583/193083801-712e838c-81b2-4cc7-a9e0-62c8d2f46021.png)

https://docs.google.com/document/d/e/2PACX-1vSaQjYm4IFt5GvLfUkaEtuaMThJVEAhOK6QVSdE-fyPUJu8ROCDPm3TLdLNJxB91_0H9NQ5ULJjlayo/pub

**Comment by piyushsahujtp** [comment]
https://docs.google.com/document/d/e/2PACX-1vSaQjYm4IFt5GvLfUkaEtuaMThJVEAhOK6QVSdE-fyPUJu8ROCDPm3TLdLNJxB91_0H9NQ5ULJjlayo/pub

this is my vlog , kindly find mistake and guide me for making changes

**Comment by EshaalakshmiDS** [comment]
@akshitagupta15june Hello ma'am this is my submission under GSSOC'24. I have also mailed you the same. I hope you consider this and assign me. 

https://docs.google.com/document/d/1pOVeefClJGBb8_Lhlq8EehTVsQur_cexLVvp1HfPrrw/edit?usp=sharing

**Comment by Darpana-iitgn** [comment]
I am interested in writing a blog. I just want to know can we get this contribution counted for hacktoberfest 2024? If yes, then pls inform me so that i can start writing.
